speaker:
  print_say: true
  speak_enabled: true
  message_level: 0
  amplitude: 10

